core/arch/arm/kernel/entry_a64.S

```diff
@@ -25,7 +25,8 @@
 		cmp	x0, #CFG_TEE_CORE_NB_CORE
 		/* Unsupported CPU, park it before it breaks something */
 		bge	unhandled_cpu
-		adr	x1, stack_tmp_stride
+		adrp	x1, stack_tmp_stride
```

Comment:
Please use `adr_l` instead.

Reply:
Sure. I was mislead by the following lines (`stack_tmp_export`) which still use `adrp`/`add` :-/ I shall convert them first.